tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// Making this a map so that if we want a specific version, the value can be the version
+var requiredPluginList = map[string]string{
+	"elasticsearch-transport":                  "",
+	"elasticsearch-api":                        "",
+	"elasticsearch":                            "",
+	"fluent-plugin-kubernetes_metadata_filter": "",
+	"fluent-plugin-concat":                     "",
+	"fluent-plugin-elasticsearch":              "",
+	"fluent-plugin-multi-format-parser":        "",
+	"fluent-plugin-record-modifier":            "",
+	"fluent-plugin-rewrite-tag-filter":         "",
+	"fluent-plugin-systemd":                    "",
+	"fluent-plugin-viaq_data_model":            "",
+	"fluent-plugin-remote-syslog":              "",
+	"fluent-plugin-prometheus":                 "",
+}
+
+// TestPluginsInstalled tests that the fluentd plugins were all installed correctly
+func TestPluginsInstalled(t *testing.T) {
+	out, err := exec.Command("fluent-gem", "list").Output()
```

Comment:
Please add a comment on where you want the test to be executed.

Reply:
:+1: